client/src/views/client-area/PaymentMethodsPage.vue

```diff
@@ -3,18 +3,60 @@
 <template>
   <div>
     <h1>PAYMENT METHODS</h1>
+    <div v-if="billingDetails">
+      <ms-stripe-card-details
+        v-if="cardDetails"
+        :card="cardDetails as MsPaymentMethod.Card"
+      />
+      {{ billingDetails }}
```

Comment:
why this line?

Reply:
Juste to display the content on the page, serves no purpose, the page is not done yet anyway.